pom.xml

```diff
@@ -34,8 +34,8 @@
     <maven.compiler.source>1.8</maven.compiler.source>
 
     <appengine.app.version>1</appengine.app.version>
-    <appengine.target.version>1.9.71</appengine.target.version>
-    <appengine-maven-plugin-version>1.3.2</appengine-maven-plugin-version>
+    <appengine.target.version>1.9.82</appengine.target.version>
+    <appengine-maven-plugin-version>2.4.0</appengine-maven-plugin-version>
```

Comment:
this update will almost certainly break the build.
v2.0.0+ require configured values for projectId and version